tools/cmake/UploadMethodConfig-Sample.cmake

```diff
@@ -0,0 +1,69 @@
+# Edit this file to configure the upload methods for your processor.
+
+# See here for documentation about these options: https://github.com/USCRPL/mbed-cmake/wiki/Upload-Methods
+
+# General config options
+# -------------------------------------------------------------
+set(UPLOAD_METHOD_DEFAULT MBED)
+
+# use a higher numbered port to allow use without root on Linux/Mac
+set(GDB_PORT 23331)
+
+# Config options for MBED
+# -------------------------------------------------------------
+
+set(MBED_UPLOAD_ENABLED TRUE)
+
+# Config options for JLINK
+# -------------------------------------------------------------
+
+set(JLINK_UPLOAD_ENABLED FALSE)
+
+if(JLINK_UPLOAD_ENABLED)
+    set(JLINK_CPU_NAME STM32F429ZI)
```

Comment:
this config contains couple of values that should be stored somewhere. The only place that comes to my mind is targets.json that would have this (MCU_ target there would contain jlink/pyocd or others). 

When back then we had project generator in exporters, that one has this in yaml files - debugging configuration for various exporters, each one has specific value and they do not match unfortunately).

Reply:
How was this used in mbed-cmake repo? This is still the main issue for this flashing bits. Based on your latest reply, would it make sense to simplify the flashing: to initially add support for mbed flashing via copy binaries and extend it with more advanced flashing like this jlink/pyOCD - as they will need further configuration. Or we just figure it out here now.